Core/Frameworks/BaikalAdmin/Controller/Install/VersionUpgrade.php

```diff
@@ -505,6 +505,13 @@ protected function upgrade($databaseConfig, $sVersionFrom, $sVersionTo) {
             $this->aSuccess[] = 'Updated default values in calendarinstances table';
         }
 
+        $config = Yaml::parseFile(PROJECT_PATH_CONFIG . "baikal.yaml");
```

Comment:
Please use a version check here like the other migrations

Reply:
Sure! Added in 35a3d01